Uzira.rte/Actors/Brains/Uzira/Uzira.lua

```diff
@@ -0,0 +1,80 @@
+function Create(self)
```

Comment:
Btw, I keep forgetting to say, but human players should be able to turn off necromancy with a pie button.